includes/classes/Indexable/Post/SyncManager.php

```diff
@@ -94,6 +96,134 @@ public function action_queue_meta_sync( $meta_id, $object_id, $meta_key, $meta_v
 		}
 	}
 
+	/**
+	 * When a term is updated, re-index all posts attached to that term
+	 *
+	 * @param  int       $term_id Term id.
+	 * @param  int       $tt_id Term Taxonomy id.
+	 * @param  string    $taxonomy Taxonomy name.
+	 * @since  3.5
+	 */
+	public function action_edited_term( $term_id, $tt_id, $taxonomy ) {
+		global $wpdb;
+
+		if ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) {
+			// Bypass saving if doing autosave
+			return;
+		}
+
+		// Find ID of all attached posts (query lifted from wp_delete_term())
+		$object_ids = (array) $wpdb->get_col( $wpdb->prepare( "SELECT object_id FROM $wpdb->term_relationships WHERE term_taxonomy_id = %d", $tt_id ) );
+
+		if ( ! count( $object_ids ) ) {
+			return;
+		}
+
+		$indexable = Indexables::factory()->get( 'post' );
```

Comment:
Minor, for improved clarity:

```suggestion
		$indexable_posts = Indexables::factory()->get( 'post' );
```

Reply:
Also for consistency with other `$indexable_*` objects we use later in the code.

Reply:
Maybe `$post_indexable` would be better? It's not actually a list of posts (which `$indexable_posts` might imply).

Reply:
Either one works for me, but we can also leave it as-is; do not consider it as blocking.